transport/http/outbound.go

```diff
@@ -282,7 +282,7 @@ func (o *Outbound) callWithPeer(
 		// maintenance loop resumes probing for availability.
 		p.OnDisconnected()
 
-		return nil, err
+		return nil, yarpcerrors.UnknownErrorf("unknown error from http client: %s", err.Error())
```

Comment:
This should be Unavailable. Indicates transient unavailability that can be resolved with retry and backoff.